### ISSUE TITLE
Keep tutorial open while play continues

### ISSUE THREAD

**Issue by jafdy** [endless-sky/endless-sky#3385]
Things like #3382 or the people who give negative steam reviews could benefit form not needing to find an obscure option in the menu to read the instructions again as they clicked out too fast thinking they where too good to need to read.
I'm thinking it is a box to the side that appears above whatever else is there and will disappear once dismissed. If possible give it a reopen button until they manage to do whatever that bit of the tutorial is about (like use the map to jump).

**Comment by KiLEdEnNis** [comment]
what if the logbook has a "player manual" in it that basically tells the player about features in the game? (controls & how to do what), and have the initial "tutorial" make that logbook known?

**Comment by KiLEdEnNis** [comment]
> and have the initial "tutorial" make that logbook known?

like mentioning the I meni

**Comment by jafdy** [comment]
But if they click strait out of it then they wouldn't even know to look there. Remember, the people this will help are the people how won't be able to work out how to get to the log book, the people who could get to the log book don't need this.

**Comment by endless-sky** [comment]
My suspicion is that a better tutorial wouldn't really improve matters for those players. Endless Sky is a game that intentionally makes the player work to figure things out and to discover new things, which some players will find enjoyable and interesting and other will just think is too hard. And anyone who doesn't even read the tutorial messages probably isn't going to like a game that involves so much reading.

**Comment by gunqqer** [comment]
Unless you force the play to take a quiz to skip the tutorial, a lot of people will skip it. And if you do it'll just piss people off. There isn't a good solution, and because all the story is reading as MZ said if they don't wanna read, not a game for them. Personaly the tutorial was very good, only thing I wish was pointed out was when I got the Cloaking Device I did relize it needed fuel to work until it was too late. 

tl;dr: only way to fix this problem is to annoy everyone, so I say we don't need much change to the current tutorials

**Comment by Nechochwen-D** [comment]
What about shipping the game with a tutorial mode that uses a smaller map (maybe 10 systems) with more detailed handholding than James currently gives.

**Comment by jafdy** [comment]
How many games do you play the tutorial of first? Most people will skip the tutorial at first and either give up (no helping them) or try and go back to it. As of now, going back to it is only possible through an out-of-the-way menu. While I agree that there can be too much hand holding, not being able to work out how to jump makes a dreadful first experience.

Say you come to a new game you have never played before (that was free), and you start it up and can't work out how to move. How long are you going to keep at trying to work that out (say the dev put movement on IJKL instead of WASD for some reason). Say you do work out that strange key binding, so you walk round a corner and die. Turns out you could have pressed M to look round the corner without getting hurt. Let's say you work this out too, and you finish the mission or whatever it was you where doing. You are now just a little bit invested in the game so you might be willing to continue working out the strange keybindings.

ES isn't a particularly challenging game to learn but getting stuck in the first system is a very fast way to loose a player.
My big problem is that the current tutorial is a pop up that doesn't let you play until you have gotten rid of it, if it was kept out of the way but still visible then people would just ignore it until they needed help, when they could read it and keep trying as many times as they want. 

A quick question; What are the most essential controls we have for being able to play the game? 
And; How obvious are these controls?

**Comment by xobes** [comment]
relates to #10133
